apps/admin/src/app/(api)/api/v1/solar/route.ts

```diff
@@ -0,0 +1,157 @@
+import { createHash } from "node:crypto";
+import { env } from "@/env.mjs";
+import { getUserById, getUserData, log, logError } from "@energyleaf/db/query";
+import { waitUntil } from "@vercel/functions";
+import { type NextRequest, NextResponse } from "next/server";
+
+interface Props {
+    userId: string;
+    userHash: string;
+    watts: number;
+}
+
+interface OpenStreetMapProps {
+    lat: number;
+    lon: number;
+    display_name: string;
+}
+
+interface BrightSkyProps {
+    weather: {
+        timestamp: string;
+        solar: number;
+    }[];
+}
+
+export const POST = async (req: NextRequest) => {
```

Comment:
Ihr werdet CORS Erros bekommen wenn ihr den Post von Web App zu Admin App macht.
Besser wäre wahrscheinlich das alles als Server Action in der WebApp